test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -62,13 +63,23 @@ call_log_manager::process_call_log()
 void
 call_log_manager::process_call_log_entry(json call_log_entry)
 {
-    const std::string method_name = call_log_entry["MethodName"].get<std::string>();
-    switch (_api_map.at(method_name)) {
-    case api_method::wiredtiger_open:
-        /* conn = &connection_simulator::get_connection(); */
-        break;
-    case api_method::open_session:
-        break;
+    try {
+        const std::string method_name = call_log_entry["method_name"].get<std::string>();
+        switch (_api_map.at(method_name)) {
+        case api_method::wiredtiger_open:
+            _conn = &connection_simulator::get_connection();
+            break;
+        case api_method::open_session:
+            break;
+        case api_method::set_timestamp:
+            std::cout << "Set timestamp call" << std::endl;
```

Comment:
I think you forgot to remove this?

Reply:
Yeah I'll remove it here and add it in the fix up ticket. It's much easier to test with it since we're no implementing the handling of the API calls in order.